includes/BasicObjects/Surface.js

```diff
@@ -1,177 +1,455 @@
+* @param	{String}	comment			Comment, can be undefined
+* @param	{Object}	params  		Surface's parameters, can be undefined
+* @returns	Created surface
+*/
 Surface.prototype.WithoutMembraneTension = function (no,
-    geometry_type,
-    geometry_type_parameters,
-    boundary_lines,
-    thickness,
-    comment,
-    params) {
-    geometry_type_parameters = typeof geometry_type_parameters !== 'undefined' ? geometry_type_parameters : [];
-    boundary_lines = typeof boundary_lines !== 'undefined' ? boundary_lines : [];
-    this.surface = engine.create_surface(no, boundary_lines);
-    this.surface.type = surfaces.TYPE_WITHOUT_MEMBRANE_TENSION;
-    this.surface.geometry = geometry_type;
-    if (thickness !== 'undefined') {
-        this.surface.thickness = thickness;
-    }
-    set_comment_and_parameters(this.surface, comment, params);
-};
-
-/**
- * Create Load Distribution Surface
- * @param {int} no - Number of Surface
- * @param {array} boundary_lines - List of boundary lines of the surface
- * @param {string} load_transfer_direction - Load transfer direction
- * @param {string} comment - Comment for the surface
- * @param {dictionary} params - Parameters of the surface
- */
-Surface.prototype.LoadDistribution = function (no,
-    boundary_lines,
-    load_transfer_direction,
-    comment,
-    params) {
-    boundary_lines = typeof boundary_lines !== 'undefined' ? boundary_lines : [];
-    this.surface = engine.create_surface(no, boundary_lines);
-    this.surface.type = surfaces.TYPE_LOAD_TRANSFER;
-    this.surface.load_transfer_direction = load_transfer_direction;
-    set_comment_and_parameters(this.surface, comment, params);
-};
\ No newline at end of file
+	boundary_lines,
+	thickness,
+	comment,
+	params) {
+	return this.surface = createBaseSurface(no, boundary_lines, surfaces.TYPE_WITHOUT_MEMBRANE_TENSION, thickness, comment, params);
+};
+
+/**
+* Creates load transfer surface
+* @param	{Number}	no				Index of surface, can be undefined
+* @param	{Array}		boundary_lines	List of boundary lines indexes
+* @param	{String}	comment			Comment, can be undefined
+* @param	{Object}	params  		Surface's parameters, can be undefined
+* @returns	Created surface
+*/
+Surface.prototype.LoadTransfer = function (no,
+	boundary_lines,
+	comment,
+	params) {
+	return this.surface = createBaseSurface(no, boundary_lines, surfaces.TYPE_LOAD_TRANSFER, undefined, comment, params);
+};
+
+/**
+* Sets plane geometry type of surface
+*/
+Surface.prototype.Plane = function () {
+	this.surface.geometry = surfaces.GEOMETRY_PLANE;
+};
+
+/**
+* Sets quadrangle geometry type of surface
+* @param {Number}	corner_node_1	Quadrangle corner 1, can be undefined
+* @param {Number}	corner_node_2	Quadrangle corner 2, can be undefined
+* @param {Number}	corner_node_3	Quadrangle corner 3, can be undefined
+* @param {Number}	corner_node_4	Quadrangle corner 4, can be undefined
+*/
+Surface.prototype.Quadrangle = function (corner_node_1,
```

Comment:
Add here also no, params and comment as I could use this to create surface and then set its type